test1/task3/main.c

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
+#include <stddef.h>
+
+int signsFrequency(char *path)
+{
+    FILE *fl = fopen(path, "r");
```

Comment:
```suggestion
    FILE *file = fopen(path, "r");
```
Не нужно сокращать без крайней необходимости.